File: apps/problems/Reverse-String/boilerplate-full/function.rs
use std::fs::read_to_string;
    use std::io::{self};
    use std::str::Lines;
    
    ##USER_CODE_HERE##
    
    fn main() -> io::Result<()> {
      let input = read_to_string("/dev/problems/Reverse-String/tests/inputs/##INPUT_FILE_INDEX##.txt")?;
      let mut lines = input.lines();
      let originalString: String = lines.next().unwrap().parse().unwrap();
      let result = reverseString(originalString);
      println!("{}", result);
      Ok(())
    }
    

File: apps/problems/Reverse-String/boilerplate-full/function.cpp
#include <iostream>
    #include <fstream>
    #include <vector>
    #include <string>
    #include <sstream>
    #include <climits>
    
    ##USER_CODE_HERE##
    
    int main() {
      std::ifstream file("/dev/problems/Reverse-String/tests/inputs/##INPUT_FILE_INDEX##.txt");
      std::vector<std::string> lines;
      std::string line;
      while (std::getline(file, line)) lines.push_back(line);
    
      file.close();
      std::string originalString;
  std::istringstream(lines[0]) >> originalString;
      std::string result = reverseString(originalString);
      std::cout << result << std::endl;
      return 0;
    }
    